pipelines/common/src/main/java/com/google/fhir/analytics/AvroConversionUtil.java

```diff
@@ -151,12 +151,29 @@ synchronized AvroConverter getConverter(String resourceType) throws ProfileExcep
     return converterMap.get(resourceType);
   }
 
+/*
   @VisibleForTesting
   @Nullable GenericRecord convertToAvro(Resource resource) throws ProfileException {
     AvroConverter converter = getConverter(resource.getResourceType().name());
     // TODO: Check why Bunsen returns IndexedRecord instead of GenericRecord.
     return (GenericRecord) converter.resourceToAvro(resource);
   }
+  */
+  
+  // Catch any RuntimeException and log the error instead of crashing the whole pipeline for an issue with one FHIR resource.
+  @VisibleForTesting
+  @Nullable GenericRecord convertToAvro(Resource resource) throws ProfileException {
+    AvroConverter converter = getConverter(resource.getResourceType().name());
+    // TODO: Check why Bunsen returns IndexedRecord instead of GenericRecord.
+	GenericRecord gr = null;
```

Comment:
Please make sure that you run `mvn clean package` and include the formatting changes after that in the PR. There are some style issues in these lines which should be fixed by that.